### PR TITLE
The end of an era

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
Removes my status as a code-owner from repository files, removes my name from the maintainer list, and also cleans up something I forgot a while back.

### CODE REVIEW

**Comment by Calyxman** [comment]
Wait what

**Comment by hal9000PR** [comment]
I just want to say. Thank you for everything you have done for paradise. I appreciate you and all the work you have done. I know many people will agree with me on this sentiment.

**Comment by PacifistDalek** [comment]
tbh not enough labels on this pr

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -20,7 +20,6 @@ dreamchecker.exe @AffectedArc07
 rust_g.dll @AffectedArc07
 
 
-
```
Comment (S34NW) [inline .github/CODEOWNERS:23]: webeditor heresy caused this, and and the best bit is you don't have to care anymore

**Review by variableundefined — APPROVED** [review]

**Pull request merged**